npm/private/npm_translate_lock.bzl

```diff
@@ -382,6 +384,13 @@ def npm_translate_lock(
 
             Read more: [lifecycles](/docs/pnpm.md#lifecycles)
 
+        lifecycle_hooks_use_default_shell_env: If True, the `use_default_shell_env` attribute of the lifecycle hooks
+            actions is set to True.
+
+            This defaults to False to reduce the negative effects of `use_default_shell_env`.
+
+            Read more: [lifecycles](/docs/pnpm.md#lifecycles)
```

Comment:
that link doesn't seem useful in this case

Reply:
I was thinking we should add to that doc?

Reply:
See the update to that doc